### DISCUSSION TITLE
Switching doesn't change audio device between 2 devices

### DISCUSSION THREAD

**Discussion by TheRealDjElite** [Belphemur/SoundSwitch#1162] [Q&A] [answered]
**Windows 11 Pro, Version 22H2, 22621.1555
SoundSwitch v6.6.0**

I have 2 sound devices check marked.

Whether I use the shortcut or from within the app, I cannot switch audio devices successfully. The audio device choice changes (even in Windows 11 sound settings), however, the actual sound source itself doesn't.

I have to go into Windows 11 sound settings and disable either audio device to kickstart the other one, then re-enable the disabled one.

Rebooting the system doesn't help either.

**Comment by canobis** [comment]
I have the same issue, happened after the recent update, sometimes the hot key will randomly switch devices but i have to hit the hot key several times. Also the volume control slider doesn't reflect the actual device that is currently set. it only reflects what windows currently has set as default. So changing the volume up or down still says 100 and doesn't visually move the slider bar even though the volume is actually changing. Did a clean reinstall and the issue still persists, rebooting also did nothing.

**Comment by nikoxley** [comment]
I was having similar issues and have rolled back to version 6.5.4.0 which seems to be OK so far.

**Comment by Belphemur** [comment, marked as answer]
Can you try the v6.6.1 and see if it fixes it ?

**Reply by TheRealDjElite** [reply]
Problem solved! 💯😁👍🏽

**Comment by canobis** [comment]
v6.6.1 fixed it. thanx!
love this app

**Comment by TheRealDjElite** [comment]
Suddenly it's happening again. 😖

**Reply by jipijajay** [reply]
I guess it's the same issue for me. 
I am not exactly sure since when but it may be it started with to 22H2. While troubleshooting I recognized this 'new' option to swtich the output. - At least for me that is new. ;)
![image](https://user-images.githubusercontent.com/94906623/236450679-a76d6d0a-cf02-4352-a5dc-628743abcbf1.png)


My Windows sound settings are no longer responding/changing with SoundSwitch. :(

**Comment by druid49** [comment]
Im having the same issue now in Oct 2025
